src/Arithmetic.elm

```diff
@@ -221,33 +247,58 @@ intCubeRoot =
 otherwise, return `Nothing`.
 
     exactIntCubeRoot 800 == Nothing
+
     exactIntCubeRoot 1000 == Just 10
+
 -}
 exactIntCubeRoot : Int -> Maybe Int
 exactIntCubeRoot n =
     let
         s =
             intCubeRoot n
     in
-        if s ^ 3 == n then
-            Just s
-        else
-            Nothing
+    if s ^ 3 == n then
+        Just s
+
+    else
+        Nothing
 
 
 {-| Test whether a number is a cube.
 
     isCube 800 == False
+
     isCube 1000 == True
+
 -}
 isCube : Int -> Bool
 isCube n =
     let
         r =
-            n % 63
+            modBy 63 n
     in
-        (r == 0 || r == 1 || r == 8 || r == 27 || r == 28 || r == 35 ||
-         r == 36 || r == 55 || r == 62) && intCubeRoot n ^ 3 == n
+    (r
+        == 0
+        || r
```

Comment:
ouch, some parentheses would help (elm-format respects those)
```elm
((r == 0)
  || (r == 1)
  || ...
)
```

Reply:
True, I'm not sure how I didn't see that! That is indeed hard to read lol

Reply:
I pushed a fix, it's not my favorite syntax but it should be easier to read.